submissions/Yasya23/holi-style-popup/index.html

```diff
@@ -0,0 +1,257 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Menu</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+  <body>
+    <nav class="navigation">
+      <ul class="navigation__list">
+        <li class="navigation__item"><a href="#">Mail</a></li>
+        <li class="navigation__item"><a href="#">Images</a></li>
+        <li class="navigation__item popup">            
+            <label for="popup"
```

Comment:
The same with formatting here
 
<img width="1216" alt="Screen Shot 2022-08-12 at 13 21 33" src="https://user-images.githubusercontent.com/21179601/184335459-37a65f92-90a0-4197-bae6-f08554abd2fa.png">